themes/bootstrap3/js/common.js

```diff
@@ -423,35 +423,11 @@ var VuFind = (function VuFind() {
    */
   var initResultScripts = function initResultScripts(container) {
     let jqContainer = typeof container === 'string' ? $(container) : container;
-    if (typeof this.doi !== 'undefined') {
-      this.doi.embedDoiLinks(jqContainer);
-    }
-    if (typeof this.openurl !== 'undefined') {
-      this.openurl.init(jqContainer);
-    }
-    if (typeof this.itemStatuses !== 'undefined') {
-      this.itemStatuses.init(jqContainer);
-    }
-    if (typeof this.saveStatuses !== 'undefined') {
-      this.saveStatuses.init(jqContainer);
-    }
-    if (typeof this.recordVersions !== 'undefined') {
-      this.recordVersions.init(jqContainer);
-    }
-    if (typeof this.cart !== 'undefined') {
-      this.cart.registerToggles(jqContainer);
-    }
-    if (typeof this.embedded !== 'undefined') {
-      this.embedded.init(jqContainer);
-    }
-    this.lightbox.bind(jqContainer);
+    emit('result-scripts', jqContainer);
```

Comment:
I don't like the specificity of the name of the event. Could something like 'results-init' be used? It's all scripts handling it anyway, but this would convey the meaning of the event better.
Also, since we're trying to reduce jQuery dependencies, let's pass the plain JS container in the event, and make the parameter an object like `{container: unwrapJQuery(container)}` so that it's easier to add more parameters later if needed.

Reply:
I renamed the event and reduced the jQuery as suggested.

Reply:
@ThoWagen I suggested `results-init` because it would align nicely with `results-load` and `results-loaded`. Now `init-result` also sounds like it could be specific to a single search result.

Reply:
That makes sense. I changed it.